.jenkins/dsl/jobs.groovy

```diff
@@ -69,48 +98,67 @@ if (!isMainBranch()) {
 def otherFolder = KogitoConstants.KOGITO_DSL_OTHER_FOLDER
 if (isMainBranch()) {
     folder(otherFolder)
-    setupOptaPlannerTurtleTestsJob(otherFolder);
+    setupOptaPlannerTurtleTestsJob(otherFolder)
 }
 
 /////////////////////////////////////////////////////////////////
 // Methods
 /////////////////////////////////////////////////////////////////
 
-void setupOptaplannerPrJob(String jobFolder) {
+void setupOptaplannerPrJob() {
     def jobParams = getDefaultJobParams()
-    jobParams.job.folder = jobFolder
-    jobParams.pr = [ blackListTargetBranches: ['7.x'] ]
+    jobParams.pr.whiteListTargetBranches = ['8.5.x', '8.7.x']
     KogitoJobTemplate.createPRJob(this, jobParams)
 }
 
-void setupOptaplannerQuarkusLTSPrJob(String jobFolder) {
+void setupOptaplannerQuarkusLTSPrJob() {
     def jobParams = getDefaultJobParams()
-    jobParams.job.folder = jobFolder
-    jobParams.pr = [ blackListTargetBranches: ['7.x'] ]
+    jobParams.pr.whiteListTargetBranches = ['8.5.x', '8.7.x']
     KogitoJobTemplate.createQuarkusLTSPRJob(this, jobParams)
 }
 
-void setupOptaplannerNativePrJob(String jobFolder) {
+void setupOptaplannerNativePrJob() {
     def jobParams = getDefaultJobParams()
-    jobParams.job.folder = jobFolder
-    jobParams.pr = [ blackListTargetBranches: ['7.x'] ]
+    jobParams.pr.whiteListTargetBranches = ['8.5.x', '8.7.x']
     KogitoJobTemplate.createNativePRJob(this, jobParams)
 }
 
-void setupOptawebEmployeeRosteringPrJob(String jobFolder) {
+void setupOptawebEmployeeRosteringPrJob() {
     def jobParams = getDefaultJobParams('optaweb-employee-rostering')
-    jobParams.job.folder = jobFolder
     jobParams.pr = [ whiteListTargetBranches: ['master'] ]
     KogitoJobTemplate.createPRJob(this, jobParams)
 }
 
-void setupOptawebVehicleRoutingPrJob(String jobFolder) {
+void setupOptawebVehicleRoutingPrJob() {
     def jobParams = getDefaultJobParams('optaweb-vehicle-routing')
-    jobParams.job.folder = jobFolder
     jobParams.pr = [ whiteListTargetBranches: ['master'] ]
     KogitoJobTemplate.createPRJob(this, jobParams)
 }
 
+void setupMultijobPrDefaultChecks() {
+    KogitoJobTemplate.createMultijobPRJobs(this, getMultijobPRConfig()) {
+        def jobParams = getDefaultJobParams()
+        jobParams.pr.blackListTargetBranches = ['7.x', '8.5.x', '8.7.x']
```

Comment:
If we don't allow the multi-job PR for these branches, what branches are actually permitted?

Reply:
`master` and all future release branches.

Reply:
`7.x` will remain as it was for the previous existing PR check. `8.5.x` and `8.7.x` will be removed once support for those branches will be removed